### ISSUE TITLE
View Website progress

### ISSUE THREAD

**Issue by hovski** [hovski/animal-rescue-website#1]
Hey @hellachella, this is the progress so far on my website. There's a few issues I'm having with the logo on the top left corner and I'm unsure how to get it to appear. Any advice would be great.

**Comment by hellachella** [comment]
Hey, just cloned and opened. The design is looking great.

You have uploaded your production files to github by accident. You need to move assets, content, design etc offline.

I don't think your logo.svg exported properly. It seems to be empty.

**Comment by hovski** [comment]
So I had issues with the logo.svg and the spritesheet for the icons at the bottom. I ended up removing the spritesheet and placing the icons individually. The logo is now the logo.png because for some reason I'd do it right, but it wouldn't save other than a white space?

I'm having severe issues with making my image have an orange hover + text. I can't even get the hover to work. I have no idea how to fix it since I tried it as a background-image and I've also done it with just an img tag through css. Nothing is working and I'm at the end of the rope here... trying to solve the issue. I have no idea how to fix it - i've tried a lot of methods.
